daemon/daemon.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/containerd/log"
 	"github.com/distribution/reference"
 	dist "github.com/docker/distribution"
+	"github.com/dustin/go-humanize"
```

Comment:
Curious if this package is consistent with `github.com/docker/go-units` which we use elsewhere. (I know it has issues, but I think we use that package in most places).